public/editor/scripts/editor/js/fc/publisher.js

```diff
@@ -173,6 +173,11 @@ define(function(require) {
     var dialog = publisher.dialog;
     var buttons = dialog.buttons;
 
+    var didConfirm = window.confirm("Are you sure you want to unpublish this project? Links that have already been shared will no longer work.");
+
+    if (!didConfirm) {
+      return;
+    }
     if (publisher.unpublishing) {
```

Comment:
Can you put this check before the `didConfirm` declaration so that we don't show it if it is already unpublishing.